cernopendata/modules/fixtures/data/records/data-policies.json

```diff
@@ -0,0 +1,120 @@
+[
+{
+  "abstract": {
+    "description": "This document contains the LHCb Data Access Policy. This was adopted at the Collaboration Board meeting on 27th Feb 2013."
+  }, 
+  "accelerator": "CERN LHC", 
+  "authors": [
+    {
+      "affiliation": "U. Edinburgh", 
+      "name": "Clarke, Peter"
+    }
+  ], 
+  "collaboration": {
+    "name": "LHCb collaboration"
+  }, 
+  "collections": [
+    "Data-Policies"
+  ], 
+  "date_published": "2013", 
+  "doi": "10.7483/OPENDATA.LHCb.HKJW.TWSZ", 
+  "experiment": "LHCb", 
+  "language": "eng", 
```

Comment:
In most other places we use "English" and not "eng" (e.g. `cms-learning-resources.json`). Would it be a quick fix to harmonise?

Reply:
Done.